todo_app/templates/index.html

```diff
@@ -8,7 +8,7 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="insertForm">
-    <form>
+    <form action = "{{ url_for('add_item') }}" method="POST"> 
       <label for = "title">Item Title</label> <br>
       <input type = "text" id ="title" name = "title"> <br> <br>
```

Comment:
Minor, but adding a `required` attribute to this input can be good by encouraging the browser to do some quick validation for us - users can still get round it if they want to, but from a user-experience point of view it's still a good start by helping them avoid accidentally creating blank to-dos!